schema/manifest_backwards_compatibility_test.go

```diff
@@ -136,14 +136,14 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 		//
 		// curl -L -H "Authorization: Bearer ..." -H \
 		// "Accept: application/vnd.docker.distribution.manifest.v2+json" \
-		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc
+		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446
```

Comment:
On Tue, Jan 03, 2017 at 05:00:57PM -0800, Zhou Hao wrote:
> `go test`
> `PASS`

I agree that the *test pass* with your 36073936, but the *point* of these tests is to show that the current OCI spec is compatible with existing Docker-hub blob instances with minimal media type replacements.  If you change the Docker-hub blobs locally, you break that.  Using the token fetching comment from #370:

    $ TOKEN=$(curl https://auth.docker.io/token\?service\=registry.docker.io\&scope\=repository:library/docker:pull  | jq -r .token)
    $ curl -sILH "Authorization: Bearer ${TOKEN}" -H "Accept: application/vnd.docker.distribution.manifest.v2+json" https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc | head -n1
    HTTP/1.1 200 OK
    $ curl -sILH "Authorization: Bearer ${TOKEN}" -H "Accept: application/vnd.docker.distribution.manifest.v2+json" https://registry-1.docker.io/v2/library/docker/manifests/sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446 | head -n1
    HTTP/1.1 404 Not Found

The tests in this backwards-compat file need to be blobs you can successfully pull from the Docker hub, not ones that 404.